src/EnergyPlus/LowTempRadiantSystem.hh

```diff
@@ -159,9 +159,11 @@ namespace LowTempRadiantSystem {
         );
 
         Real64 setRadiantSystemControlTemperature();
-
+        
+        Real64 calculateOperationalFraction(Real64 const offTemperature, Real64 const controlTemperature, Real64 const throttlingRange);
+        
         virtual void calculateLowTemperatureRadiantSystem(ZoneTempPredictorCorrectorData &dataZoneTempPredictorCorrector, Real64 &LoadMet) = 0;
-
+                
```

Comment:
Your editor left some trailing white space on this line, no worries for now, but I'd check to see if you can tell it to remove them upon saving the file.